pgjdbc/src/test/resources/pg_service/pgservicefileEnv.conf

```diff
@@ -0,0 +1,20 @@
+
+# comment
+[test-service1]
+  host=pgservicefileEnv-test-host.test.net
+        port=5433
+   # ürdivõi ööjää
```

Comment:
What is this ?

Reply:
test case for service entry, spaces in front of key=value paris and comment with non-ascii (expected to be couple of correct unicode) characters